hadoop-common-project/hadoop-auth/pom.xml

```diff
@@ -272,5 +272,68 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>java-8</id>
+      <activation>
+        <jdk>1.8</jdk>
+      </activation>
```

Comment:
Why are these profiles being added in `hadoop-common-project/hadoop-auth/pom.xml` instead of the root POM?

1. We may need the `--add-opens` args for other tests.
2. More importantly, the compiler settings should be global.

Reply:
> Why are these profiles being added in `hadoop-common-project/hadoop-auth/pom.xml` instead of the root POM?
> 
> 1. We may need the `--add-opens` args for other tests.
> 2. More importantly, the compiler settings should be global.

@adoroszlai Sure I will move it to hadoop-main pom.